dbmail/utils.py

```diff
@@ -1,17 +1,23 @@
 # -*- encoding: utf-8 -*-
 
-from django.utils.html import strip_tags
+import logging
 
 from dbmail import import_module
+from django.utils.html import strip_tags
+
+logger = logging.getLogger(__name__)
 
 
 def premailer_transform(text):
     try:
         from premailer import transform
 
         return transform(text)
+    except ImportError:
```

Comment:
Why do you catch ImportError here? The next block will do the same.

Reply:
Next block catch all other errors that can be in transform() method. 
Catching ImportError here for case if premailer is not installed. For example, I don't want to install premailer because of dependencies or package version incompatibility.